pubsub/subscriptions.js

```diff
@@ -84,6 +84,32 @@ function createSubscription (topicName, subscriptionName) {
 }
 // [END pubsub_create_subscription]
 
+// [START pubsub_subscriber_flow_settings]
+function createFlowControlledSubscription (topicName, subscriptionName, maxInProgress, maxBytes) {
```

Comment:
I guess the Pub/Sub samples were never updated to hide the method definition?

Reply:
Correct - I can address that in a separate PR, if you'd like.

Reply:
Yes please